### PR TITLE
Fixed the timezone CSS bug

### LINKED ISSUE #529: The buttons and links shift downwards when the input field of "select timezone" is selected
**Actual Behaviour**

The right part of the page gets dislocated and the buttons and links shift downwards when the input field of select timezone is selected in the account option of settings.

**Expected Behaviour**

The page shouldn't get dislocated when the option is selected.

**Steps to reproduce it**

Go to settings/account. Try to select a timezone.

**Screenshots of the issue**

**Would you like to work on the issue?**

 Yes.

### PR DESCRIPTION
**Pull request by plxity**
Fixes #529 

Changes: On using timezone dropdown the button and link don't shift downwards

Surge Deployment Link: https://pr-636-fossasia-susi-accounts.surge.sh

capture

### CODE REVIEW

**Comment by plxity** [comment]
@anshumanv @akshatnitd  Please Review this.

File: src/components/Settings/Settings.css [hunk 1]
```diff
@@ -257,3 +257,10 @@ input[type=file]{
     width: 70%;
   }
 }
+.time-zone{
+   padding-bottom: 30px
```
Comment (akshatnitd) [inline src/components/Settings/Settings.css]: Use `;` at the end.

Comment (plxity) [reply src/components/Settings/Settings.css]: @akshatnitd  Done!

**Comment by plxity** [comment]
@isuruAb  Thank you!

**Pull request merged**